### DISCUSSION TITLE
Unhandled exception. System.IO.FileNotFoundException:

### DISCUSSION THREAD

**Discussion by wkmhv** [unoplatform/uno#18727] [General]
vs 2022 17.11.5
"msbuild-sdks": {
  "Uno.Sdk": "5.5.32"
},
"sdk":{
  "allowPrerelease": false
}
Visual Studio keeps getting these errors

[INFO] Project information hasn't been loaded, retrying...
[INFO][08:02:49.21] Uno Remote Control initialized (5.5.37+Branch.release-stable-5.5.Sha.8f536cafcafb4c6c1d11ae441e7226be20f4cfd9.8f536cafcafb4c6c1d11ae441e7226be20f4cfd9)
[INFO] Loaded the Uno.WinUI.DevServer service (5.5.37, API v2).
[ERROR][08:02:57.90] Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'. Das System kann den angegebenen Pfad nicht finden.
[ERROR][08:02:57.90] File name: 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'
[ERROR][08:02:57.90]    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
[ERROR][08:02:57.90]    at System.Reflection.Assembly.LoadFrom(String assemblyFile)
[ERROR][08:02:57.90]    at Uno.UI.RemoteControl.Helpers.AssemblyHelper.Load(IImmutableList`1 dllFiles, Boolean throwIfLoadFailed) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Helpers/AssemblyHelper.cs:line 21
[ERROR][08:02:57.90]    at Uno.UI.RemoteControl.Host.Extensibility.AddInsExtensions.ConfigureAddIns(IWebHostBuilder builder, String solutionFile) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Extensibility/AddInsExtensions.cs:line 13
[ERROR][08:02:57.90]    at Uno.UI.RemoteControl.Host.Program.Main(String[] args) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Program.cs:line 87
[ERROR][08:02:57.90]    at Uno.UI.RemoteControl.Host.Program.<Main>(String[] args)
[ERROR][08:02:58.56] 
[ERROR][08:03:09.69] Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'. Das System kann den angegebenen Pfad nicht finden.
[ERROR][08:03:09.69] File name: 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'
[ERROR][08:03:09.69]    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
[ERROR][08:03:09.69]    at System.Reflection.Assembly.LoadFrom(String assemblyFile)
[ERROR][08:03:09.69]    at Uno.UI.RemoteControl.Helpers.AssemblyHelper.Load(IImmutableList`1 dllFiles, Boolean throwIfLoadFailed) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Helpers/AssemblyHelper.cs:line 21
[ERROR][08:03:09.69]    at Uno.UI.RemoteControl.Host.Extensibility.AddInsExtensions.ConfigureAddIns(IWebHostBuilder builder, String solutionFile) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Extensibility/AddInsExtensions.cs:line 13
[ERROR][08:03:09.69]    at Uno.UI.RemoteControl.Host.Program.Main(String[] args) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Program.cs:line 87
[ERROR][08:03:09.69]    at Uno.UI.RemoteControl.Host.Program.<Main>(String[] args)
[ERROR][08:03:10.14] 
[ERROR][08:03:19.41] Unhandled exception. System.IO.FileNotFoundException: Could not load file or assembly 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'. Das System kann den angegebenen Pfad nicht finden.
[ERROR][08:03:19.41] File name: 'C:\Users\Werner KÃ¶nig\.nuget\packages\uno.settings.devserver\1.0.25\tools\devserver\Uno.Settings.DevServer.dll'
[ERROR][08:03:19.41]    at System.Runtime.Loader.AssemblyLoadContext.LoadFromAssemblyPath(String assemblyPath)
[ERROR][08:03:19.41]    at System.Reflection.Assembly.LoadFrom(String assemblyFile)
[ERROR][08:03:19.41]    at Uno.UI.RemoteControl.Helpers.AssemblyHelper.Load(IImmutableList`1 dllFiles, Boolean throwIfLoadFailed) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Helpers/AssemblyHelper.cs:line 21
[ERROR][08:03:19.41]    at Uno.UI.RemoteControl.Host.Extensibility.AddInsExtensions.ConfigureAddIns(IWebHostBuilder builder, String solutionFile) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Extensibility/AddInsExtensions.cs:line 13
[ERROR][08:03:19.41]    at Uno.UI.RemoteControl.Host.Program.Main(String[] args) in /agent/_work/1/s/src/Uno.UI.RemoteControl.Host/Program.cs:line 87
[ERROR][08:03:19.41]    at Uno.UI.RemoteControl.Host.Program.<Main>(String[] args)
[ERROR][08:03:19.87]

**Comment by jeromelaban** [comment]
Thanks for the report, we'll take a look!

**Comment by dr1rrb** [comment]
@wkmhv able to repro on my side by adding some accentuated chars in my nuget path. Looking to fix that right now, but FYI you should be able to workaround the issue by adding a `nuget.config` file in the root dir of your project and redirect the nuget path to something simpler:

```xml
<?xml version="1.0" encoding="utf-8"?>
<configuration>
    <config>
        <add key="globalPackagesFolder" value="C:\Nuget" />
        <add key="repositoryPath" value="C:\Nuget" />
    </config>
</configuration>

```

**Reply by dr1rrb** [reply]
FYI here is the fix https://github.com/unoplatform/uno/pull/18733

**Comment by wkmhv** [comment]
Thanks for the quick reply. Works perfectly
